modules/administration-guide/partials/proc_disabling-readiness-init-containers-for-the-olm-installer.adoc

```diff
@@ -0,0 +1,44 @@
+:_module-type: PROCEDURE
+
+[id="disabling-readiness-init-containers-for-the-olm-installer_{context}"]
+= Disabling the readiness init containers for the OLM installer
+
+To disable the previously enabled readiness init containers for {prod-short} installed by the OLM installer:
+
+.Prerequisites
+
+* {prod-short} is installed by the OLM installer.
+
+.Procedure
+
+. Find the {prod-short} Operator subscription name:
++
+[source,shell,subs="+quotes,+attributes"]
+----
+$ {orch-cli} get subscriptions -n {prod-namespace}
+----
+
+. Get the CSV (Cluster Service Version) name from the {prod-short} Operator subscription:
```

Comment:
```suggestion
. Find the CSV (Cluster Service Version) name in the {prod-short} Operator subscription:
```

Reply:
Just adding the above suggestion in case you think we'd better use `find` instead of `get` in this particular step.